### ISSUE TITLE
Be explicit about empty environment

### ISSUE THREAD

**Issue by 00vareladavid** [JuliaLang/Pkg.jl#745]
Running `status` on an empty environment currently looks like this:
```
(MyPackage) pkg> st
Project MyPackage v0.1.0
    Status `/tmp/MyPackage/Project.toml
```

What about:
```
(MyPackage) pkg> st
Project MyPackage v0.1.0
    Status `/tmp/MyPackage/Project.toml
    Empty environment.
```

It doesn't matter to me much what the message is. Just that we are explicit. This is mainly to help new users, but it doesn't hurt experienced users either.

**Comment by StefanKarpinski** [comment]
Yes, seems like a good change to me.

**Comment by 00vareladavid** [comment]
Solved by #748
